Assets/VRTK/Scripts/VRTK_AdaptiveQuality.cs

```diff
@@ -0,0 +1,708 @@
+    /// </example>
+    public sealed class VRTK_AdaptiveQuality : MonoBehaviour
+    {
+        #region Public fields
+
+        [Tooltip("Toggles whether the render quality is dynamically adjusted to maintain VR framerate.\n\n"
+                 + "If unchecked, the renderer will render at the recommended resolution provided by the current VRDevice.")]
+        public bool active = true;
+
+        [Tooltip("Toggles whether to show the debug overlay.\n\n"
+                 + "Each square represents a different level on the quality scale. Levels increase from left to right,"
+                 + " and the first green box that is lit above represents the recommended render target resolution provided by"
+                 + " the current VRDevice. The yellow boxes represent resolutions below the recommended render target resolution.\n"
+                 + "The currently lit box becomes red whenever the user is likely seeing reprojection in the HMD since the"
+                 + " application isn't maintaining VR framerate. If lit, the box all the way on the left is almost always lit red because"
+                 + " it represents the lowest render scale with reprojection on.")]
+        public bool drawDebugVisualization;
+
+        [Tooltip("Toggles whether to allow keyboard shortcuts to control this script.\n\n"
+                 + "The supported shortcuts are:\n"
+                 + "- Shift+F1: Toggle debug visualization on/off\n"
+                 + "- Shift+F2: Toggle usage of override render scale on/off\n"
+                 + "- Shift+F3: Decrease override render scale level\n"
+                 + "- Shift+F4: Increase override render scale level")]
+        public bool respondsToKeyboardShortcuts = true;
+
+        [Tooltip("Toggles whether to allow command line arguments to control this script at startup of the standalone build.\n\n"
+                 + "The supported command line arguments all begin with \"-\" and are:\n"
+                 + CommandLineArguments.Disable + ": Disable adaptive quality\n"
+                 + CommandLineArguments.MinimumRenderScale + " X: Set minimum render scale to X\n"
+                 + CommandLineArguments.MaximumRenderScale + " X: Set maximum render scale to X\n"
+                 + CommandLineArguments.MaximumRenderTargetDimension + " X: Set maximum render target dimension to X\n"
+                 + CommandLineArguments.RenderScaleFillRateStepSizeInPercent + " X: Set render scale fill rate step size in percent to X (X from 1 to 100)\n"
+                 + CommandLineArguments.OverrideRenderScaleLevel + " X: Set override render scale level to X\n"
+                 + CommandLineArguments.DrawDebugVisualization + ": Enable debug visualization\n"
+                 + CommandLineArguments.MSAALevel + " X: Set MSAA level to X")]
+        public bool respondsToCommandLineArguments = true;
+
+        [Header("Quality")]
+        [Range(0, 8)]
+        [Tooltip("The MSAA level to use.")]
```

Comment:
In other scripts I had put the tooltip before the other options (like range). Not sure if it's worth doing the same here for consistency (i'm not that fussed though)